reportwidgets/NewUsers.php

```diff
@@ -0,0 +1,128 @@
+<?php namespace RainLab\User\ReportWidgets;
+
+use Backend\Classes\ReportWidgetBase;
+use Carbon\Carbon;
+use Exception;
+use Lang;
+use RainLab\User\Models\User;
+
+class NewUsers extends ReportWidgetBase
+{
+    /**
+     * Render the widget.
+     */
+    public function render()
+    {
+        try {
+            $this->loadData();
+        } catch (Exception $e) {
+            $this->vars['error'] = $e->getMessage();
+        }
+
+        return $this->makePartial('widget');
+    }
+
+    /**
+     * Define widget properties.
+     * 
+     * @return array
+     */
+    public function defineProperties()
+    {
+        return [
+            'widget_title' => [
+                'default'           => Lang::get('rainlab.user::lang.widgets.new_users.default_title'),
```

Comment:
Could you double check that the call to Lang::get is required here? I could be wrong but I think that October already passes the default through Lang::get

Reply:
Sure. I'm pretty sure I already tried that, but I'll double check tonight when I get off work.

Reply:
Confirmed, this property requires manual translation.